client/js/components/procedure/DpSimplifiedNewStatementForm.vue

```diff
@@ -42,7 +42,7 @@
             <dp-radio
               name="r_role"
               value="0"
-              data-cy="simplifiedNewStatementForm:citizenButton"
+              :data-cy="`roleInput:${Translator.trans('citizen').toLowerCase()}`"
```

Comment:
please don't use translators in your hooks. the translations can vary between the projects.

Reply:
https://github.com/demos-europe/demosplan-core/pull/3280/commits/5408e907a06aa66448e126ffad3806bf14b6f5f8